.travis.yml

```diff
@@ -3,7 +3,6 @@ sudo: required
 install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
 script: bash -ex .travis-opam.sh
 env:
-   - OCAML_VERSION=4.02
```

Comment:
This chooses the wrong version of core_kernel that doesn't have the splittable_random functionality.
Eventually I'll bring that inside this library so it doesn't include a core dependency but for now we'll keep it.